openpathsampling/netcdfplus/netcdfplus.py

```diff
@@ -714,6 +718,17 @@ def create_type_delegate(self, var_type):
         setter = None
         store = None
 
+        if var_type.startswith('obj.') or var_type.startswith('lazyobj.'):
+            store = getattr(self, var_type.split('.')[1])
+            base_type = store.content_class
+
+            get_iterable = lambda v: \
+                not v.base_cls is base_type if hasattr(v, 'base_cls') else hasattr(v, '__iter__')
+
+            set_iterable = lambda v: \
+                not v.base_cls is base_type if hasattr(v, 'base_cls') else hasattr(v, '__iter__')
```

Comment:
Why `get_iterable` and `set_iterable`? Would these be better named something like `setter_is_iterable`? Starting a name with `set` sounds like the function will set an attribute.